src/cls/IPM/Repo/Http/PackageService.cls

```diff
@@ -0,0 +1,97 @@
+Include %IPM.Common
+
+Class %IPM.Repo.Http.PackageService Extends (%RegisteredObject, %IPM.Repo.IPackageService)
+{
+
+Parameter ALLOWPREFIXINLOCATION = 1;
+
+Property Location As %String;
```

Comment:
Nit: If I'm understanding correctly, this can be set to the %IPM.DataType.RepoLocation datatype like the URL IPM.Repo.Http.Definition.cls